tools/topology/topology1/sof-imx8ulp-btsco-16k.m4

```diff
@@ -0,0 +1,85 @@
+#
+# Topology for i.MX8ULP board with bluetooth codec
+#
```

Comment:
For such a simple modification we don't need to create a new `m4` file. We can go with a paramter.

e.g

```
"sof-imx8ulp-btsco\;sof-imx8ulp-btsco\;-DRATE=8000"
"sof-imx8ulp-btsco\;sof-imx8ulp-btsco-16k\;-DRATE=16000"
```

and then something like this:


```
PIPELINE_PCM_ADD(sof/pipe-volume-capture.m4,
	2, 0, 1, s16le,
	1000, 0, 0,
	`RATE`, `RATE`, `RATE`)
```

Reply:
This makes code clean. Will change it. Thanks!